src/components/experiment-wrapper/experiment-wrapper.js

```diff
@@ -39,37 +48,28 @@ const ExperimentWrapper = ({ theme }) => {
   const [newRunAdded, setNewRunAdded] = useState(false);
   const [isDisplayingMetrics, setIsDisplayingMetrics] = useState(false);
 
-  // Reload state is to ensure it will call useRedirectLocationInExperimentTracking
-  // only when the component is re-rendered or reloaded.
-  const [reload, setReload] = useState(false);
+  const [enableComparisonView, setEnableComparisonView] = useState(false);
+  const [selectedRunIds, setSelectedRunIds] = useState([]);
+  const [activeTab, setActiveTab] = useState(tabLabels[0]);
+  const [errorMessage, setErrorMessage] = useState({});
+  const [invalidUrl, setInvalidUrl] = useState(false);
+  const [usedNavigationBtn, setUsedNavigationBtn] = useState(false);
 
-  useEffect(() => setReload(true), []);
+  const history = useHistory();
```

Comment:
I don't think this is being used anywhere.

Reply:
removed it now 👍